README.md

```diff
@@ -1,5 +1,16 @@
 # PyPSA-Earth-Sec: A Sector-Coupled Open Optimisation Model of the Global Energy System
 
-## Development Status: Under Development
+## Development Status: Version 0.0.5
```

Comment:
Why version 0.0.5? We could start with 0.0.1? Or 0.1?

Reply:
Agree, in my opinion we could start with 0.0.1